.github/workflows/periodic.yml

```diff
@@ -0,0 +1,37 @@
+name: Periodic CI
+
+on:
+  schedule:
+    - cron: '11 7 * * 1,4'
```

Comment:
This repo sees so infrequent changes I think its fair to only run this once a month or once every 2 weeks instead of twice a week

Reply:
true, but does it cost us anything to run it more regularly?

Reply:
Just because we don’t pay for it does not mean its free :-D